app/controllers/citizens/dependant_details_controller.rb

```diff
@@ -1,35 +1,37 @@
 module Citizens
-  class DependantDetailsController < BaseController
-    include ApplicationFromSession
-    helper_method :other_dependants
+  module Dependants
+    class DetailsController < BaseController
```

Comment:
I think this controller needs to be removed - it's been superseded by the file at `app/controllers/citizens/dependants/details_controller.rb`